assets/css/components/_buttons.scss

```diff
@@ -0,0 +1,11 @@
+.flex-button-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  > * {
```

Comment:
It’d be great to avoid wildcard selectors like this, in favor of applying CSS classes directly to the items we want to style. These types of selectors feel concise, but get unwieldy quickly.